offocus/src/java/org/jivesoftware/openfire/plugin/ofmeet/JitsiJicofoWrapper.java

```diff
@@ -86,7 +86,7 @@ public synchronized void initialize() throws Exception
         // Disable JVB rediscovery. We are running with one hard-coded videobridge, there's no need for dynamic detection of others.
         System.setProperty( "org.jitsi.jicofo.SERVICE_REDISCOVERY_INTERVAL", "-1" ); // Aught to use a reference to ComponentsDiscovery.REDISCOVERY_INTERVAL_PNAME, but that constant is private.
 
```

Comment:
I've created https://github.com/igniterealtime/ofmeet-openfire-plugin/issues/14 to track this specific change.